### ISSUE TITLE
Wrong datatype in conllparser error message

### ISSUE THREAD

**Issue by shepdl** [huggingface/neuralcoref#241]
If a coref has an error in it (for example, `'label'`, `'start'` or `'end'` is `None`), the assertion will fail, but the code that builds the assertion error message itself has an error. The lines (conllparser.py:352-356) are as follows:

```python
assert (
                coref["label"] is not None
                and coref["start"] is not None
                and coref["end"] is not None
            ), ("Error in coreference " + coref + " in " + parsed)
```

Because `coref` is a dict, not a str, this will produce the error `TypeError: can only concatenate str (not "dict") to str`.

I can submit a pull request, but I'm not sure what the best information to include in the error message is.

**Comment by svlandeg** [comment]
We'd definitely welcome a pull request! In general it's best to include some information in the error message that helps the user identify the original culprit, i.e. as close to the original input as possible, without printing too much additional overhead.

**Comment by svlandeg** [comment]
Also, feel free to use f-strings, as we don't have to support Python 2 anymore!

**Comment by svlandeg** [comment]
Fixed by PR #242 - thanks again @shepdl !
